### DISCUSSION TITLE
Ender 3 V3 KE Support

### DISCUSSION THREAD

**Discussion by Rulevoid** [ballaswag/creality_k1_klipper_mod#1] [Q&A]
Most of the other K1 install scripts work great on the V3 KE. I'd love to get my KE over to mainline Klipper and asked on Reddit if this would work. Word at that point is that it should, but I noticed in the install script that it looks at the machine info to identify the model of K1 and the else statement just errors. So it wouldn't work out of the box for the KE without tweaking.  

Has anyone actually ran through and tried this? If it works, it would be awesome to see official support added for it.

**Comment by Jbake90** [comment]
I'm pulling my hair out trying to go back and adapt this install for the V3 KE, i wish someone could do this asap
